src/portals/hope/features/dashboard/Greeting.tsx

```diff
@@ -12,9 +12,11 @@ const getDayPartGreeting = (name: string) => {
   } else if (hours >= 18 && hours <= 23) {
     return `Good evening ${name}!`
   }
+
+  return ''
```

Comment:
I guess something is better than nothing
```suggestion
  return `Hello ${name}`
```